plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TableSnapshot.java

```diff
@@ -182,6 +187,15 @@ public List<Transaction> getTransactions()
         return logTail.getTransactions();
     }
 
+    public long getRetainedSizeInBytes()
```

Comment:
Is `cachedMetadata` excluded intentionally?

Reply:
added now